docs/new-dependency-policy.md

```diff
@@ -0,0 +1,31 @@
+# New Package Dependency Policy
+
+This is the policy for adding a new package dependency to the Cardano SL suite of package.
+
+1. Whenever a new dependency is added to an existing Cardano SL package that fact should be
+   noted in the Github Pull Request (PR).
+
+2. If the new dependency is already a direct or indirect dependency of an existing
+   Cardano SL package that should be noted in the Github PR. Existing dependencies do
+   not need further veting.
+
+3. If the new dependency is not already a direct or indirect dependency of an existing
+   Cardano SL package the following information should be noted in the PR:
+
+   * The Hackage entry for the new package.
+   * What functionality the new package provides for Cardano SL.
+   * The level of risk the new package provides.
+
+Assessing the level of risk should be relatively easy:
+
+1. Is the package only used for testing/benchmarking (low risk).
+2. Does the package provide cryptographic functionality or is it network or blockchain related
+   (high risk).
+3. Is it a widely used, well maintained package from a well known Haskeller or from an someone
+   employed by IOHK (low risk).
+4. Is the package obscure, not widely used from someone that is not a well known Haskeller (high
+   risk).
```

Comment:
I would add:

5. Does the package have decent test coverage?
6. Does the package have decent documentation coverage? Does the package have a tutorial?
7. Are the package maintainers responsive to issues and PRs? Are there any old issues or PRs that haven't been addressed?